src/Calculator/Resources/en-US/Resources.resw

```diff
@@ -4715,4 +4715,8 @@
     <value>Memory list</value>
     <comment>Automation name for the group of controls for memory list.</comment>
   </data>
+  <data name="Format_HistorySlotCleared" xml:space="preserve">
+    <value>History slot %1 cleared</value>
```

Comment:
Consider changing slot to item.

Reply:
hi @sanderl.  I left it as 'slot' to have parity with Memory item announcements.  If you feel 'item' suits better I will change it.

Reply:
Makes sense to be synced with memory to stay consistent, will mark this thread as resolved with the current changes.